oem.json

```diff
@@ -3297,21 +3603,52 @@
         "description": "ADB: Package Protected.\nMy Q Paks \nThird-party application bundles\nhttps://www.financialmirror.com/2007/10/31/motorola-packs-moto-q-9h-global-smart-device-with-third-party-applications/\n",
         "removal": "delete"
     },
+    {
+        "id": "com.motorola.personalize",
+        "label": "Personalise",
+        "description": "Helps you personalise your themes, icons, fonts, sounds etc...",
+        "removal": "delete"
+    },
     {
         "id": "com.motorola.pgmsystem2",
         "description": "Appears safe to remove\nPGM System\nI didn't find info about this package. \nFor Me PGM = Peak Gate Power (for MOSFET transistor) but I'm not convinced it has this meaning here.\n",
         "removal": "delete"
     },
+    {
+        "id": "com.motorola.photoeditor",
+        "label": "Moto Photo Editor",
+        "description": "On Motorola phones that feature Cutout mode, you can replace the background of your photos and resize & move selected portions.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.photoeditor"
+        ],
+        "removal": "delete"
+    },
     {
         "id": "com.motorola.programmenu",
-        "description": "Programming Menu\nHidden menu (accessible by typing  ##7764726 in the dialer) providing additionnal features for developers.\n",
+        "label": "Programming Menu",
+        "description": "Hidden menu (accessible by typing  ##7764726 in the dialer) providing additionnal features for developers.",
         "removal": "delete"
     },
     {
         "id": "com.motorola.ptt.prip",
         "description": "Prip (https://play.google.com/store/apps/details?id=com.motorola.ptt.prip)\nPush-To-Talk app. Allows to you send calls over any wireless carrier’s 3G or 4G networks or a WiFi connection.\nIt offers unlimited calling between other users and Nextel phone owners, rather than universal calling credit, \nand works on a monthly subscription basis.\nhttps://prip.me/#get\nNo longer in Android 10 image\n",
         "removal": "delete"
     },
+    {
+        "id": "com.motorola.screenshoteditor",
+        "label": "Screenshot editor",
+        "description": "Moto default screenshot app that supports longer screenshots & gif.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.screenshoteditor"
+        ],
+        "removal": "replace"
+    },
+    {
+        "id": "com.motorola.settings",
+        "label": "System update",
+        "description": "Exactly not sure what it does",
```

Comment:
Does it break OTA update or something?

Reply:
On my testing OTA update checker works fine without this.
I'll still assign this as `caution`.
> description : Show privacy policy, legal info after your 1st boot or device updates.